### PR TITLE
Correct the ROS and Fortress bridge

### LINKED ISSUE #429: Wrong ROS2 bridge in Fortress ros integration documentation
## Environment
* OS Version: Ubuntu 20204 
* Binary build: Gazebo Sim, version 6.16.0

## Description
In the [documentation of the bridge between Gazebo and ROS2](https://github.com/gazebosim/docs/blob/master/fortress/ros2_integration.md#bidirectional-communication) It is written to use the command `ros2 run ros_ign_bridge parameter_bridge /TOPIC@ROS_MSG@IGN_MSG` to create the bridge, which is wrong as you will get the error that `Package 'ros_ign_bridge' not found`.
The solution is to use `ros2 run ros_gz_bridge parameter_bridge` instead.

* Expected behavior:   The command in the documentation should work!
* Actual behavior:  Package 'ros_ign_bridge' not found

## Steps to reproduce

1. Install Gazebo Fortress following [Ubuntu Binary documentation](https://gazebosim.org/docs/fortress/install_ubuntu)
2. Install ROS 2 Humble following their [Ubuntu doc](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html)
3. Install Gazebo ROS2 pairing following [Gazebo doc](https://gazebosim.org/docs/harmonic/ros_installation#installing-the-default-gazebo-ros-pairing)
4. Run the command mentioned in [ROS2 integration doc publish key stroke](https://gazebosim.org/docs/fortress/ros2_integration#publish-key-strokes-to-ros)

## Output
`Package 'ros_ign_bridge' not found`

## Solution
Use `ros2 run ros_gz_bridge parameter_bridge`

### PR DESCRIPTION
**Pull request by Mohammadreza-Beygifard**
# 🦟 Bug fix

Fixes #429

## Summary

I am updating the documentation of the ROS2 <-> Gazezebo fortress bridge. By mistake in the documentation the command `ros_ign_bridge` was used; However, the correct command to create the bridge is `ros2 run ros_gz_bridge parameter_bridge`. For more details, please refer to the issue.

## Checklist
- [ ] Signed all commits for DCO
- [ ] Added tests
- [x] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [x] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [x] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: fortress/ros2_integration.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ In this tutorial we will learn how to Integrate ROS 2 with Ignition. We will est
 communication between them. This can help in many aspects; we can receive data or commands
 from ROS and apply it to Ignition and vice versa.
 
-## ros_ign_bridge
+## ros_gz_bridge
 
-`ros_ign_bridge` provides a network bridge which enables the exchange of messages between ROS 2 and Ignition Transport. Its support is limited to only certain message types. Please, check this [README](https://github.com/ignitionrobotics/ros_ign/blob/ros2/ros_gz_bridge/README.md) to verify if your message type is supported by the bridge.
+`ros_gz_bridge` provides a network bridge which enables the exchange of messages between ROS 2 and Ignition Transport. Its support is limited to only certain message types. Please, check this [README](https://github.com/ignitionrobotics/ros_ign/blob/ros2/ros_gz_bridge/README.md) to verify if your message type is supported by the bridge.
```
Comment (azeey) [inline fortress/ros2_integration.md:9]:
```suggestion
`ros_gz_bridge` provides a network bridge which enables the exchange of messages between ROS 2 and Ignition Transport. Its support is limited to only certain message types. Please, check this [README](https://github.com/gazebosim/ros_gz/blob/ros2/ros_gz_bridge/README.md) to verify if your message type is supported by the bridge.
```

**Review by azeey — APPROVED** [review]
LGTM! I only have one minor comment.

Also, can you fix the conflict?

**Comment by Mohammadreza-Beygifard** [comment]
@azeey  I have addressed the conflict and everything looks good now, please inform me if there is anything else I can do. Best!

**Review by azeey — APPROVED** [review]

**Comment by pxalcantara** [comment]
@azeey How are you? just to understand the process. When will be made the next deploy of the documentation and this update will be implemented?

@Mohammadreza-Beygifard congratulations and thanks for your contribution, I had the same problem!

**Pull request merged**